google/cloud/forseti/scanner/scanners/iap_scanner.py

```diff
@@ -292,7 +292,7 @@ def make_iap_resource(self, backend_service, project_full_name):
         direct_access_sources = set()
         for backend in backend_service.backends:
             instance_group = self.find_instance_group_by_url(
-                backend.get('group'))
+                backend.get('resourceGroup'))
```

Comment:
Please make the change in the CAI inventory, rather than changing the scanner.

Reply:
Updated, PTAL!

Reply:
Thanks for helping Joe!